dsgrid/config/project_config.py

```diff
@@ -182,12 +182,13 @@ class InputDatasetModel(DSGBaseModel):
     )
     # TODO this model_sector must be validated in the dataset_config
     # TODO: this is only necessary for input_model types
-    model_sector: str = Field(  # TODO: should this be data_source instead?
+    model_sector: str = Field(  # TODO: should this be data_source instead? -- still relevant?
         # TODO: need to discuss with team why this is needed. One potential reason is because it is
         # helpful to query at some point which datasets have not yet been registered. Dataset_id may
         # not be all that descriptive, but the data_source is. We may also want the metric_type here too.
         title="model_sector",
         description="Model sector ID, required only if dataset type is ``sector_model``.",
+        # -- still relevant?
         optional=True,
         updateable=False,
     )
```

Comment:
Is all of this still relevant if we make the switch to "modeled" instead of "sector_model"?

Reply:
Does this currently have to map to a sector listed in the project base dimensions? If so, that's probably overly strict and we should drop it. But if it's just a loose tag, it might be nice to keep.